source/Halibut.Tests/TestServices/EchoService.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.IO;
+using System.Threading;
+using Halibut.TestUtils.Contracts;
+
+namespace Halibut.Tests.TestServices
+{
+    public class EchoService : IEchoService
```

Comment:
@nathanwoctopusdeploy In `Halibut.Tests` we have the async versions of services.

I don't think this is used, and was supposed to be deleted (as we have `AsyncEchoService` here, and `EchoService` in `Halibut.TestUtils.CompatBinary.Base`)

Reply:
Deleted!